src/style.css

```diff
@@ -1 +1,66 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Zen+Antique+Soft&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  background: #fff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-align: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+}
+
+.nav_item {
+  display: inline-block;
+  position: relative;
+  height: 34px;
```

Comment:
no need
try never to use height explicitly (it's a bad practice), only in cases when you haven't got another way